content/SCALE/SCALETutorials/SystemSettings/Services/OpenVPNSCALE.md

```diff
@@ -4,14 +4,16 @@ description: "Provides configuration information for OpenVPN Client and Server s
 weight: 30
 alias: 
 tags:
- - scaleopenvpn
+- scaleopenvpn
 ---
 
 
 {{< toc >}}
 
 {{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
 
+To configure VPN service, go to **Apps > Available Applications** and locate the **[wg-easy]({{< relref "InstallWgeasyApp.md" >}}** application.
```

Comment:
missing close parentheses 
```suggestion
To configure VPN service, go to **Apps > Available Applications** and locate the [**wg-easy**]({{< relref "InstallWgeasyApp.md" >}}) application.
```

Reply:
Not sure if this renders correctly so I'm going to apply bold outside of the [ ].